### DISCUSSION TITLE
As anyone tried to create a Confluence Plugin for Excalidraw ?

### DISCUSSION THREAD

**Discussion by clainchoupi** [excalidraw/excalidraw#3705] [General]
Hi guys, in my team we are using Excalidraw as much as we can, and we were thinking of an integration with Confluence.

Do you know if someone tried to make an Excalidraw Plugin ?
I think I will try to initiate it, and maybe see if it is possible to remove the collaborative features (just use the drawing features for the plugin)

**Comment by dwelle** [comment]
:+1:

I'm not aware of a plugin for Confluence yet, though there was some interest (I guess you've seen https://github.com/excalidraw/excalidraw/issues/1189).

**Reply by clainchoupi** [reply]
I had not! Really interesting thread, I'll have a deeper look 🙂
